src/keylessBackup/saga.test.ts

```diff
@@ -269,9 +305,13 @@ describe('keylessBackup saga', () => {
             [call(generateKeysFromMnemonic, mockMnemonic), { privateKey: mockPrivateKey }],
             [call(walletHasBalance, privateKeyToAddress(mockPrivateKey)), true],
             [call(assignAccountFromPrivateKey, mockPrivateKey, mockMnemonic), mockWalletAddress],
+            [call(initializeAccountSaga), undefined],
+            [call(storeMnemonic, mockMnemonic, mockWalletAddress), undefined],
+            [
+              call(storeSECP256k1PrivateKey, mockEncryptionPrivateKeyHex, mockWalletAddress),
+              undefined,
+            ],
```

Comment:
We should leave the `.call` if we want to use these functions are called. These don't assert that these functions were called, just provides mocks. If for example initializeAccountSaga is removed, this test will still pass.